ReactWindows/ReactNative.Shared.Tests/ReactNative.Shared.Tests.csproj

```diff
@@ -40,6 +40,10 @@
     <PlatformTarget>ARM</PlatformTarget>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Facebook.CSSLayout, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Facebook.CSSLayout.1.1.1\lib\portable-net45+dnxcore50+win+wpa81+wp80+MonoAndroid10+xamarinios10+MonoTouch10\Facebook.CSSLayout.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```

Comment:
The Facebook.CSSLayout version we're using is 2.0.1-pre, but it doesn't look like you're using this right now anyway.